packages/clarity-js/src/data/metadata.ts

```diff
@@ -25,18 +25,10 @@ export function start(): void {
     pageNum: s.count
   }
 
-  // For backward compatibility; remove in future iterations (v0.6.11)
-  if (config.upload && typeof config.upload === Constant.String && (config.upload as string).indexOf(Constant.HTTPS) === 0) {
-    let url = config.upload as string;
```

Comment:
Just checking that use of this old version is negligible before you remove this.

Reply:
Yes confirmed. Thanks for pointing out.